From: Miguel Vadillo <vadillo@ti.com>
Date: Wed, 9 Feb 2011 22:27:40 +0000 (-0600)
Subject: syslink: ipu_pm: remove unused variable
X-Git-Tag: android-2.6.35-2.3-omap4.13.1~86
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=687b388e4e9404d826984f2c51d47c6a4ee0f721

syslink: ipu_pm: remove unused variable

MAX_IPU_COUNT is not needed in the code and can
be removed.

Change-Id: Ifed37474d88fe4df8861f0e1ec37941b6da36319
Reported-by: Paul Hunt <hunt@ti.com>
Signed-off-by: Miguel Vadillo <vadillo@ti.com>
---

diff --git a/drivers/dsp/syslink/ipu_pm/ipu_pm.h b/drivers/dsp/syslink/ipu_pm/ipu_pm.h
index 4ed30f4..787ecc6 100644
--- a/drivers/dsp/syslink/ipu_pm/ipu_pm.h
+++ b/drivers/dsp/syslink/ipu_pm/ipu_pm.h
@@ -100,12 +100,6 @@
 #define APP_M3 1
 #define TESLA 0
 
-/* If sysm3 or appm3 is requested ipu will be automatically requested
- * this is beacause the cstrs can only be set to ipu and not individually.
- * SYSM3 + APPM3 + IPU
- */
-#define MAX_IPU_COUNT 3
-
 #define PM_CSTR_PERF_MASK	0x00000001
 #define PM_CSTR_LAT_MASK	0x00000002
 #define PM_CSTR_BW_MASK		0x00000004
